packages/scss/src/components/inputs/textfield/_textfield.framed.scss

```diff
@@ -0,0 +1,186 @@
+		width: 100%;
+
+		&:focus, &:hover {
+			z-index: 1;
+
+			~ .textfield-messages {
+				transform: translateY(100%);
+			}
+		}
+	}
+
+	.textfield-label {
+		font-size: _theme("sizes.small.font-size");
+		left: _component("textfield.framed.side-padding");
+		right: _component("textfield.framed.side-padding");
+		top: _component("textfield.framed.label-top-offset");
+		width: auto;
+	}
+
+	.textfield-suffix {
+		top: _component("textfield.framed.top-padding");
+		right: _component("textfield.framed.side-padding");
+	}
+
+	.textfield-messages {
+		bottom: 0;
+		left: 0;
+		pointer-events: none;
+		position: absolute;
+		right: 0;
+		text-align: center;
+		transform: translateY(0);
+		transition: transform _theme("commons.animations.durations.standard");
+		z-index: 1;
+	}
+
+	.textfield-messages-error {
+		background-color: _color("error");
+		color: _theme("palettes.error.text");
+		font-size: _component("textfield.message.font-size");
+		margin: 0;
+	}
+
+	.textfield-messages-warning {
+		background-color: _color("warning");
+		color: _theme("palettes.warning.text");
+		font-size: _component("textfield.message.font-size");
+		margin: 0;
+	}
+
+	.textfield-messages-helper {
+		background-color: _color("text.light");
+		color: white;
+		font-size: _component("textfield.message.font-size");
+		margin: 0;
+	}
+
+	// COLORING
+
+	@mixin inputColoring($palette) {
+		.textfield-input:focus, .textfield-input:hover {
+			@include fakeBorder(1px, _get($palette, "color"));
+
+			background-color: transparentize(_get($palette, "color"), .95);
+			~ .textfield-label {
+				color: _get($palette, "color");
+			}
+		}
+	}
+
+	@include inputColoring(_component("textfield.framed.default-palette"));
+
+	// palette colors
+	@each $name, $palette in _palettes() {
+		&.palette-#{$name} {
+			@include inputColoring($palette);
+		}
+	}
+
+
+	// MODS
+	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+	&.mod-multiline {
+		max-width: none;
+		padding: 1rem;
+		@include fakeBorder(1px, _theme("commons.divider.color"));
+
+		textarea.textfield-input {
+			height: _component("textfield.framed.textarea.height");
+			padding: 0;
+			margin-top: _component("textfield.framed.textarea.margin-top");
+			box-shadow: none;
+			resize: none;
+		}
+
+		// PALETTE
+		// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
+
+		@mixin inputColoring($palette) {
+			&:hover, &:focus {
+				@include fakeBorder(1px, _get($palette, "color"));
+				background-color: transparentize(_get($palette, "color"), .95);
+				z-index: 2;
+			}
+
+			textarea.textfield-input {
+				&:focus, &:hover {
+					box-shadow: none;
+					background: transparent;
+					z-index: auto;
+				}
+			}
+		}
+
+		@include inputColoring(_component("textfield.compact.default-palette"));
+
+		// palette colors
+		@each $name, $palette in _palettes() {
+			&.palette-#{$name} {
+				@include inputColoring($palette);
+			}
+		}
+
+		// MODS
```

Comment:
We need to specify a style for `.textfield-suffix` here (.mod-framed.mod-multiline) that pushes it to the bottom right corner.
Maybe :
top: auto;
bottom: XXem;